test/unit/specs/components/Identity/getIdentity/createGetIdentity.spec.js

```diff
@@ -0,0 +1,51 @@
+import createGetIdentity from "../../../../../../src/components/Identity/getIdentity/createGetIdentity";
+
+describe("Identity::createGetIdentity", () => {
+  let sendEdgeNetworkRequest;
+  let createIdentityPayload;
+  const samplePayload = {
+    myPayload: { methodOne: () => {}, methodTwo: () => {} }
+  };
+  beforeEach(() => {
+    sendEdgeNetworkRequest = jasmine.createSpy("sendEdgeNetworkRequest");
+    createIdentityPayload = jasmine
+      .createSpy("createIdentityPayload")
+      .and.returnValue(samplePayload);
+  });
+  it("should return a function which calls sendEdgeNetworkRequest", () => {
+    const getIdentity = createGetIdentity({
+      sendEdgeNetworkRequest,
+      createIdentityPayload
+    });
+    expect(typeof getIdentity).toBe("function");
+    getIdentity();
+    expect(sendEdgeNetworkRequest).toHaveBeenCalledWith({
+      payload: samplePayload,
+      action: "identity/acquire"
+    });
+  });
+
+  it("Each getIdentity call should create a new payload object", () => {
+    const payload1 = { type: "payload1" };
+    const payload2 = { type: "payload2" };
+    createIdentityPayload.and.returnValues(payload1, payload2);
+    const getIdentity = createGetIdentity({
+      sendEdgeNetworkRequest,
+      createIdentityPayload
+    });
+    expect(typeof getIdentity).toBe("function");
+    getIdentity({ options: ["optionOne", "optionTwo"] });
```

Comment:
Won't the real call to the `getIdentity` function will look more like `getIdentity(["namespaceOne", "namespaceTwo"]);`? If so, I'd try to mimic reality here, unless there's a reason you created an object with an `options` property?